addons/l10n_in_gstin_status/models/res_partner.py

```diff
@@ -34,6 +34,9 @@ def _onchange_l10n_in_gst_status(self):
 
     def action_l10n_in_verify_gstin_status(self):
         self.ensure_one()
+        self.check_access('write')
+        if not self.env.company.sudo().account_fiscal_country_id.code != 'IN':
+            raise UserError(_('You must be logged in an Indian company to verify GSTIN status'))
```

Comment:
Isn't it

```suggestion
            raise UserError(_('You must be logged on an Indian company to verify GSTIN status'))
```
or

```suggestion
            raise UserError(_('You must be logged in in an Indian company to verify GSTIN status'))
```
?

Reply:
Based on the current occurrences, I'll go for: `You must be logged in an Indian company to use this feature`. Is that ok for you ?

```
~/src/enterprise$ ack -h 'You must be logged' --py | sed "s/^\s*//;s/'/\"/g;s/[.]\"/\"/g;" | sort | uniq -c
    247 raise UserError(_("You must be logged in a Belgian company to use this feature"))
     16 raise UserError(_("You must be logged in a Hong Kong company to use this feature"))
     21 raise UserError(_("You must be logged in a Indian company to use this feature"))
     24 raise UserError(_("You must be logged in a Kenyan company to use this feature"))
     11 raise UserError(_("You must be logged in a Luxembourger company to use this feature"))
      5 raise UserError(_("You must be logged in an Australian company to use that feature"))
      2 raise UserError(_("You must be logged in an Egypt company to use this feature"))
      8 raise UserError(_("You must be logged in a %s company to use this feature", country_restriction))
     32 raise UserError(_("You must be logged in a Swiss company to use this feature"))
      2 raise UserError(_("You must be logged in a U.A.E. company to use this feature"))
     14 raise UserError(_("You must be logged in a US company to use this feature"))
      3 raise UserError(_("You must be logged into a Belgian company to print the individual account"))
      2 raise UserError(_("You must be logged into a Saudi Arabian company to use this feature"))
      8 raise UserError(_("You must be logged into a Swiss company to use this feature"))
     11 raise UserError(_("You must be logged in to process some of these work orders"))
      3 "You must be logged in a %(country_code)s company to use this feature",
```

Reply:
Ya, oki ! I don't know why that sentence makes my brain bug haha